webpack/assets/javascripts/react_app/components/common/forms/Form.stories.js

```diff
@@ -78,14 +79,6 @@ storiesOf('Components/Form', module)
         <Row>
           <FormField {...ownComponentFieldProps} />
         </Row>
-        <Row>
-          <FormField
-            {...ownComponentFieldProps}
-            label="Customly rendered field"
-          >
-            <CustomSelectWithContext firstOption="CustomlyRenderedInput" />
```

Comment:
If this is deleted why keep the component definition?

Reply:
You're right, thanks for noticing.